src/main/java/org/fcrepo/storage/ocfl/DefaultOcflObjectSession.java

```diff
@@ -423,6 +423,11 @@ public void versionMessage(final String message) {
         versionInfo.setMessage(message);
     }
 
+    @Override
+    public void invalidateCache(final String resourceId, final String versionNumber) {
+        headersCache.invalidate(cacheKey(resourceId, versionNumber));
```

Comment:
Actually, I think what you'd actually want to do here is only take the resource id and invalidate all headers in the cache for that id.

Reply:
Thanks @pwinckles for the review. I went this route and updated the interface to only take a `resourceId` + used `listVersions` to find the versions to invalidate.